### PR TITLE
Improve deCONZ documentation

### PR DESCRIPTION
**Pull request by Kane610**
## Proposed change

This could very well fit in with 2021.10 release but I put it to Next as siren platform is only available from 2021.10

## Type of change

- [x] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [x] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [ ] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_integrations/deconz.markdown [hunk 1]
```diff
@@ -78,6 +81,10 @@ logger:
 
 If you are having issues and want to report a problem, always start with making sure that you're on the latest [deCONZ software version](https://github.com/dresden-elektronik/deconz-rest-plugin/releases) and [latest firmware for hardware](http://deconz.dresden-elektronik.de/deconz-firmware/?C=M;O=D).
 
+### No state updates
+
+If the state of entities are only reflected in Home Assistant when the integration is loaded (during restart, reload, setup) you probably have an issue with the websocket configuration where your deCONZ instance is running. The deCONZ integration uses the websocket port provided by the deCONZ REST API. If you're running the deCONZ Docker container make sure that it properly configures the websocket port so deCONZ can report what port is exposed outside of the containerized environment. Also make sure to review firewall rules that might block communication over certain ports.
```
Comment (frenck) [inline source/_integrations/deconz.markdown]:
```suggestion
If the state of entities are only reflected in Home Assistant when the integration is loaded (during restart, reload, setup) you probably have an issue with the WebSocket configuration where your deCONZ instance is running. The deCONZ integration uses the WebSocket port provided by the deCONZ REST API. If you're running the deCONZ Docker container make sure that it properly configures the WebSocket port so deCONZ can report what port is exposed outside of the containerized environment. Also, make sure to review firewall rules that might block communication over certain ports.
```

File: source/_integrations/deconz.markdown [hunk 2]
```diff
@@ -163,23 +170,7 @@ Navigate to **Developer tools->Events**. In the section **Listen to events** add
...
-- Trådfri Open/Close Remote
-- Trådfri Remote Control
-- Trådfri Wireless Dimmer
-- Aqara Double Wall Switch
-- Aqara Mini Switch
-- Aqara Round Switch
-- Aqara Square Switch
-- Aqara Magic Cube
-- Aqara Opple 2/4/6 button Switches
+Review [this list](https://github.com/home-assistant/core/blob/15a8f6741bdf852b6f542be61aacb4896cd9e1ff/homeassistant/components/deconz/device_trigger.py#L559) to see if your device is supported by device triggers
```
Comment (frenck) [inline source/_integrations/deconz.markdown]: Ouch, I think we should not link end-user into code to figure something out

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_integrations/deconz.markdown (hunk 1 above)
Comment (Kane610) [reply source/_integrations/deconz.markdown]: Great thanks @frenck !

File: source/_integrations/deconz.markdown (hunk 2 above)
Comment (Kane610) [reply source/_integrations/deconz.markdown]: I'll simplify

**Review by frenck — APPROVED** [review]
Thanks, @Kane610 👍 

Happy Hacktoberfest 🎉 

../Frenck

**Comment by Kane610** [comment]
Happy Hacktoberfest @frenck 🎉

**Pull request merged**